cve_bin_tool/checkers/openjpeg2.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/python3
+
+"""
+CVE checker for openjpeg2
+
+https://www.cvedetails.com/product/22664/Openjpeg-Openjpeg.html?vendor_id=12064
+
+"""
+from . import Checker
+
+
+class Openjpeg2Checker(Checker):
+    CONTAINS_PATTERNS = []
+    FILENAME_PATTERNS = [r"libopenjp2\.so", r"openjpeg2"]
+    VERSION_PATTERNS = [r"openjpeg2 ([0-9]+\.[0-9]+\.[0-9]+)"]
+    VENDOR_PRODUCT = [("openjpeg2")]
```

Comment:
VENDOR_PRODUCT has to be two items (a vendor and a product) in order to work.

If you search for openjpeg in something like cvedetails, it looks like it's not openjpeg2 in the database:

https://www.cvedetails.com/google-search-results.php?q=openjpeg&sa=Search

so you'll need to fix that.